docs/docs/configuration/object_detectors.md

```diff
@@ -1032,22 +1032,21 @@ python3 yolo_to_onnx.py -m yolov7-320
 
 #### YOLOv9
 
-YOLOv9 models can be exported using the below code
+YOLOv9 model can be exported as ONNX using the below command:
 
 ```sh
-git clone https://github.com/WongKinYiu/yolov9
-cd yolov9
-
-# setup the virtual environment so installation doesn't affect main system
-# NOTE: Virtual environment must be using Python 3.11 or older.
-python3 -m venv ./
-bin/pip install -r requirements.txt
-bin/pip install onnx onnxruntime onnx-simplifier>=0.4.1
-
-# download the weights
-wget -O yolov9-t.pt "https://github.com/WongKinYiu/yolov9/releases/download/v0.1/yolov9-t-converted.pt" # download the weights
-
-# prepare and run export script
-sed -i "s/ckpt = torch.load(attempt_download(w), map_location='cpu')/ckpt = torch.load(attempt_download(w), map_location='cpu', weights_only=False)/g" ./models/experimental.py
-bin/python3 export.py --weights ./yolov9-t.pt --imgsz 320 --simplify --include onnx
+docker build . --output . -f- <<'EOF'
```

Comment:
I'm not sure I like this approach, it makes the download of yolov9-t easier but generally makes it more complicated for users to install other models if they want to try different sizes.

It could make more sense to just have a script that takes in a model size (`tiny`, `small`, `medium`) and chooses the correct weights

Reply:
I got it. You probably meant that there were more places to change the model size than before.

I fixed this, users just need to replace MODEL_SIZE now.

Reply:
I improved the description.

Reply:
> It's all one giant block now without comments, IMO it's less readable / easy to understand what is going on.

The previous comments were subtle.

Previously, users were typing in each command by themselves in their host machine, so it only made sense they knew what each command was doing.

This is no longer a concern now that it's isolated from the host machine.

For example, the big `venv` comment is no longer relevant.

I could restore the `# download the weights`, and `# prepare and run export script` commands, but as explained above I don't think these comments are needed anymore as users aren't running them on their own.